translation/translator.go

```diff
@@ -81,38 +81,60 @@ func (t *Translator) Get(key string, options ...translationcontract.Option) (str
 	// simply load the file and return the line if it exists.
 	// If the file doesn't exist, we will return fallback if it is enabled.
 	// Otherwise, we will return the key as the line.
-	if err := t.load(folder, locale); err != nil && err != ErrFileNotExist {
-		return "", err
-	}
-
-	line := t.loaded[folder][locale][keyPart]
-	if line == "" {
-		fallbackFolder, fallbackLocale := parseKey(t.GetFallback())
-		// If the fallback locale is different from the current locale, we will
-		// load in the lines for the fallback locale and try to retrieve the
-		// translation for the given key.If it is translated, we will return it.
-		// Otherwise, we can finally return the key as that will be the final
-		// fallback.
-		if (folder+locale != fallbackFolder+fallbackLocale) && fallback {
-			var fallbackOptions translationcontract.Option
-			if len(options) > 0 {
-				fallbackOptions = options[0]
+	if err := t.load(locale, folder); err != nil && err != ErrFileNotExist {
```

Comment:
Q: Why is `ErrFileNotExist` specific?

Reply:
I wanted to call fallback if there is error related to file not exists else it will return key instead of calling fallback,